Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -1830,6 +1830,65 @@
     tags:
     - VimPilot
 
+- type: entity
+  name: frog with horns
+  parent: SimpleMobBase
```

Comment:
just change `SimpleMobBase` to `MobFrog` instead of Ctrl+C Ctrl+V... I don't think a frog with horns should differ in characteristics from a regular frog, and the lack of parents makes the yml files unnecessarily huge

Reply:
I fixed it